packages/aws-cdk-lib/aws-ec2/lib/launch-template.ts

```diff
@@ -221,6 +221,17 @@ export interface LaunchTemplateProps {
    */
   readonly launchTemplateName?: string;
 
+  /**
+   * A description for the first version of the launch template.
+   *
+   * The version descrioption must be maximum 255 characters long.
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-launchtemplate.html#cfn-ec2-launchtemplate-versiondescription
+   *
+   * @default - The version description is not set if not specified.
```

Comment:
I think it could be simpler.

```suggestion
   * @default - no description
```